core/controllers/topics_and_skills_dashboard.py

```diff
@@ -167,6 +167,25 @@ def get(self):
         self.render_json(self.values)
 
 
+class SkillAssignmentsHandler(base.BaseHandler):
+    """Provides information about which topics contain the given skill."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_topics_and_skills_dashboard
+    def get(self, skill_id):
+        """Handles GET requests."""
+        topic_assignments = skill_services.get_all_topics_assigned_to_skill(
```

Comment:
@aks681 made a good point in his last comment. You'll probably want to rename the method to get_all_topic_assignments_for_skill() as well. Make any other fixes in this PR if needed.

(In general, when you get a comment and you agree with it, please think of the broader ramifications too.)

Reply:
Yeah, right.
Done.